deploy/role.yaml

```diff
@@ -44,3 +44,22 @@ rules:
   verbs:
   - create
   - get
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - "*"
```

Comment:
`get` and `create` could be enough. to be tried.

Reply:
Yes. Fixed in https://github.com/redhat-developer/devconsole-operator/pull/31/commits/d83160159c2d7d05b018051d6e22408359e952a6

Reply:
Done in https://github.com/redhat-developer/devconsole-operator/pull/31/commits/d83160159c2d7d05b018051d6e22408359e952a6